### PR TITLE
All the changes required to get a sharded K8s online

### PR DESCRIPTION
**Pull request by cberkhoff**
I'm working on an improved Readme but that will be done as a separate PR.

* Added scripts to do some common actions more easily or actions that are tripled. 
* Sorted out DNS settings. Helpers end up with names of the form: `h1-helper-shard-0.h1-helper-shard.default.svc.cluster.local`
* Created a new value `shardCount` that controls the number of shards and shard ports.
* Added an additional Deployment for the ReportCollector so that it can reach the other helpers within the same K8s network.
* Configs and Secrets conform to the new sharded configuration file structure.
* Updated LOG level

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
-helperImage: "ghcr.io/private-attribution/ipa/ipa-helper:berkhoff-sd-mt-ra"
-externalPort: 443
-port: 3000
-rustLog: "debug,h2=error"
+helperImage: "ghcr.io/private-attribution/ipa/ipa-helper:main"
+rcImage: "ghcr.io/private-attribution/ipa/rc:main"
+mpcPort: 443
+shardPort: 1443
+shardCount: 1
+rustLog: "debug,tower_http=error,hyper_util=error,rustls=error,h2=error"
```
Comment (akoshelev) [inline values.yaml:6]: is there a way to configure this at runtime? I worry about debug logging slowing down IPA execution significantly.

**Review by akoshelev — APPROVED** [review]
changes to readme would be helpful, but I'd generally prefer adding comments to bash/py files explaining their purpose

File: values.yaml (hunk 1 above)
Comment (cberkhoff) [reply values.yaml:6]: Yeah, this is just the initial, overridable value

**Comment by cberkhoff** [comment]
They had some helps in them but can be improved. Let me know. I'll also see what would be a good idea

**Pull request merged**